File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int idx=0;
        int i=0;
        while(i<nums.length-1){
            while(i<nums.length-1 && nums[i]==nums[i+1]){
                i++;
            }
            nums[idx]=nums[i];
            idx++;
            i++;
        }
        if(i==nums.length)return idx;
        nums[idx]=nums[i];
        return idx+1;
    }
}

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        x = 1
        for i in range(len(nums)-1):
            if(nums[i]!=nums[i+1]):
                nums[x] = nums[i+1]
                x+=1
        return(x)